### ISSUE TITLE
How can I handle big file upload?

### ISSUE THREAD

**Issue by robert-zaremba** [improbable-eng/grpc-web#240]
It seams that there is not direct support for big blobs of data in protobufs. The workaround is to use streaming, as explained in this [stackoverflow post](https://stackoverflow.com/questions/34969446/grpc-image-upload)

How can I uses the form multi-part file handlers in the gRPC?

**Comment by robert-zaremba** [comment]
Thanks for response. My question is only about transferring from frontend to backend. So for the multi-part upload (fronted -> backend) I will need to go with a standard HTTP request and http server on the side of the grpc one?

**Comment by johanbrandhorst** [comment]
Yes, you can add a handler on the server side that manages the fileupload separate from the gRPC server. You can even use the same port for both servers.

**Comment by maltegrosse** [comment]
@robert-zaremba see https://github.com/grpc/grpc-web/issues/517#issuecomment-629609101
